### ISSUE TITLE
Please add docs comments to functions and classes

### ISSUE THREAD

**Issue by MichalKalita** [arastta/arastta#221]
Hi,
I have problem to understand same functions, what they are doing.

For example [catalog/model/catalog/product.php](https://github.com/arastta/arastta/blob/master/catalog/model/catalog/product.php)
What is function 'getProfile($, $)' doing? Which informations contains returned data?

Source code with comments is better readable.
Advantage of commented source is possibility to automatical generate api. Etc. tool [Apigen](http://www.apigen.org)
Exists any rules to maximal length of line? Or more than 2k chars is normal? [Line with 2574 chars](https://github.com/arastta/arastta/blob/master/catalog/model/catalog/product.php#L15)

Please write comments and data types if you will edit same source.

Michal Kalita

**Comment by denisdulici** [comment]
You're right Michal, this is a known issue brought from Arastta's precursor, OpenCart and our development team is planning to fix it asap.
